astroquery/astrometry_net/tests/test_astrometry_net_remote.py

```diff
@@ -0,0 +1,143 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+import os
+
+# performs similar tests as test_module.py, but performs
+# the actual HTTP request rather than monkeypatching them.
+# should be disabled or enabled at will - use the
+# remote_data decorator from astropy:
+import pytest
+
+from astropy.tests.helper import remote_data
+from astropy.table import Table
+from astropy.io import fits
+
+from .. import conf, AstrometryNet
+from ...exceptions import TimeoutError
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
+
+
+def data_path(filename):
+    return os.path.join(DATA_DIR, filename)
+
+
+api_key = conf.api_key or os.environ.get('ASTROMETRY_NET_API_KEY')
```

Comment:
@keflavich - you've reviewed already. Did you set up an API key to run these tests, too?